Kanchan-Wakchaure/SER517-Group2-Fall2019
# Create event form connected to database

To test: 
1. Click on Create Event on the homepage.
2. Fill out the form and click on send
3. Verify data submitted to database.
